base/libc.jl

```diff
@@ -77,6 +77,17 @@ flush_cstdio() = ccall(:jl_flush_cstdio, Void, ())
 @unix_only systemsleep(s::Real) = ccall(:usleep, Int32, (UInt32,), round(UInt32,s*1e6))
 @windows_only systemsleep(s::Real) = (ccall(:Sleep, stdcall, Void, (UInt32,), round(UInt32,s*1e3)); return Int32(0))
 
+type TimevalStruct
+   sec::Int64
+   usec::Int64
```

Comment:
Are these the right size?  In the [GNU C library](http://www.gnu.org/software/libc/manual/html_node/Elapsed-Time.html), `usec` is a `long int`, which is usually 32 bits.